src/main/java/com/thealgorithms/misc/TwoSumProblem.java

```diff
@@ -0,0 +1,28 @@
+package com.thealgorithms.misc;
+
+import java.util.*;
+
+public class TwoSumProblem {
```

Comment:
```suggestion
public final class TwoSumProblem {
    private TwoSumProblem() {
    }
```

Reply:
why did you add the final keyword here?

Reply:
The class `TwoSumProblem` has only `static` methods. It does not make any sense to derive any class from it - that it is why I recommend to mark it as `final`.

Please note that I also suggest to mark the _default_ constructor as `private`.